about.html

```diff
@@ -0,0 +1,154 @@
+            </div>
+            <div class="highlevel-section-div mt-5 p-0">
+                <p style="color: #ec5242; font-size: 1.3rem;line-height: fit;"><b>"Hello! Sharing world"</b></p>
+            </div>
+            <div class="highlevel-section-div mb-5"><h2 style="color: #ec5242; font-weight:900">Creative COMMONS<br>GLOBAL SUMMIT 2015</h2></div>
+            <div class="highlevel-section-div p-3" style="text-align: center; background-color: #ffffff; border: 1px solid #d3d3d3;"><p style="font-weight: 700; font-size: 1rem; color: #272a31;">
+                The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two. This year, we hope to expand our invitation list including organizations and individuals.
+            </p></div>
+            <div class="highlevel-section-div mt-4 mb-4" style="text-align: center;"><p style="font-weight: 700; font-size: 1rem; color: #272a31;">
+                Please contact us per Email for any further questions about CC Global Summit 2015!<br>
+                <b style="text-align: center; text-decoration: underline; font-weight: 700; font-size: 1rem; color: #272a31;">ccsumit2015@cckorea.org</b>
+            </p>
+        </div>
+        </main>
+        <section id="mainprogram-section" class="highlevel-section pt-5 pb-4">
+            <div class="highlevel-section-div" style="display: flex; flex-direction: column; color: #272a31; font-size: 1.5rem; font-weight: 700;">
+                <p style="text-align: center; ">Creative Commons<br>Global Summit 2015 Logo</p><hr style="border: 1px solid #ec5242; width: 10%; margin-top: 0%;">
+                </div>
+    
+                <div class="highlevel-section-div mt-2 mb-4">
+                    <p class="p-2" style="text-align: center; color: #272a31; font-size: 0.9rem; font-weight: 700;">The logo of CC Global Summit 2015 was decided through the logo competition from 8.June to 7.July.</p>
+                    </div>
+                <div class="highlevel-section-div" style="border: 1px solid #d3d3d3"><img src="./assets/LogoTWOH.PNG" alt="The Way of Humanity" />
+                    
+                </div>
+        </section>
+        <hr style="border: 1px solid #d3d3d3; width: 100%">
+        <section id="featuredspeakers-section" class="highlevel-section">
+            <div class="highlevel-section-div mb-2" style="display: flex; flex-direction: column; color: #272a31; font-size: 1.5rem;">
+                <p style="font-weight: 700;">See the past CC Global Summits</p><hr style="border: 1px solid #ec5242; width: 10%; margin-top: 0%;">
+                </div>
+                
+                <div class="highlevel-section-div mb-5">
+                    <p style="text-align: center; font-weight: 700;">Take a look at the last two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+                    </div>
```

Comment:
- [ ] I would suggest using the `section` tags instead of `div` for `highlevel-section-div mb-5` class. The` div element` has no special meaning. It is often used as a block of children elements. The `section `element introduced in HTML5 standard is used to group related elements, such as a subsection of a long article. In short, the `section `element provides more semantic syntax than the` div` element.

Reply:
hmm.<img src="https://media2.giphy.com/media/nof1MaPOp25OCL5sYK/giphy.gif"/>